packages/astro/src/core/render/core.ts

```diff
@@ -153,3 +168,57 @@ export async function renderPage(mod: ComponentInstance, ctx: RenderContext, env
 
 	return response;
 }
+
+/**
+ * Checks whether any value can is serializable.
+ *
+ * A serializable value contains plain values. For example, `Proxy`, `Set`, `Map`, functions, etc.
+ * are not serializable objects.
+ *
+ * @param object
+ */
+export function isValueSerializable(value: unknown): boolean {
```

Comment:
these two functions might be good to put into the util module `src/core/util.ts`.

Reply:
They were in that module fore, but unfortunately `utils` imports a function from the `vite` module, which will trigger rollup errors in some parts of our application.

Reply:
Ok, that makes sense, disregard this comment.